samples/OutputBindingSamples/AddProductExtraColumns.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.OutputBindingSamples
+{
+    public static class AddProductExtraColumns
+    {
+        [FunctionName("AddProductExtraColumns")]
```

Comment:
I don't think we should be adding a binding that is wrong - even if that's on purpose. 

What if we added it to the Test project itself?